integration_test/tests/static_page_test.dart

```diff
@@ -0,0 +1,93 @@
+
+
+import 'package:flutter/material.dart';
+import 'package:flutter_markdown/flutter_markdown.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+
+import '../commons/test_util.dart';
+import '../pages/onBoarding_page.dart';
+import '../pages/setting_page.dart';
+
+void main() async{
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  group("static page test", (){
+    testWidgets("Release notes", (tester) async{
+
+      await initAppAutonomy(tester);
+      await launchAutonomy(tester);
+      await onboardingSteps(tester);
+      await addDelay(3000);
+
+      final releaseNotes = find.text("Release notes");
+      await settingScrollToFinder(tester, releaseNotes, Scrollable);
+      await tester.pump();
+      await tester.pump();
+      await addDelay(5000);
+      expect(releaseNotes, findsOneWidget);
+
+      Finder versionFinder = find.byKey(const Key("version"));
```

Comment:
Can you put the block of this code to a function to handle the string with params like the String  key, string index to delimit, int index to start for subString,...
It can make the reusable is better.